lib/Sources/Tabs/TabGridModel.swift

```diff
@@ -42,7 +43,13 @@ public final class TabGridModel {
         openTab(tab)
     }
 
-    func newTabButtonTapped() {
+    func createNewTabButtonTapped() {
+        let nextIndex = (tabs.keys.max() ?? 0) + 1
+        tabs[nextIndex] = TabModel(index: nextIndex, url: URL(string: "about:blank")!)
```

Comment:
I'm guessing this manifests as an empty webview with this URL in the serach bar when the user goes to one of these new tabs. For a better UX it might be nice to detect this case and remain on the fresh-page-view instead? Should be easy to experiment with by throwing a check for `url.isAboutBlank()` into the model here https://github.com/JuulLabs/topaz/blob/main/lib/Sources/App/WebLoadingModel.swift#L21

I didin't try it out, but please do and if you think it is better lets roll with that.